### ISSUE TITLE
BanHammer Hanging on Startup

### ISSUE THREAD

**Issue by CyanLights** [grandwazir/BanHammer#36]
Hi there again,

Our server seems to be encountering an issue where BH hangs at startup. I enabled debug and started the server, it hangs while "Enabling [the] ebean database".

We are running bukkit 2320 (1.3.1).

[Edit] It can also be noted that this is a larger server, and that our bans are somewhere above 500. I don't know if this makes a difference.

**Comment by grandwazir** [comment]
This is most likely due to the database connection timing out. Check that your database settings are correct and your firewall is not blocking it.

**Comment by grandwazir** [comment]
500 ban should not cause this issue, I personally run with around at number and do not have issues. Could you explain what your database setup is?

**Comment by CyanLights** [comment]
It is a normal MySQL database. We usually configure it in each plugin's configuration file (mcMMO, Survival Games). Is there another place that we need to configure it? I saw a section for it in bukkit.yml.

Currently, there is a BanHammer.db file in the BanHammer folder, so I assumed it was using some kind of SQL lite and not our main database.

**Comment by grandwazir** [comment]
Yes, there is documentation on how to do this. Could you disable other plugins then paste the output of the console log with debugging on?

**Comment by CyanLights** [comment]
As it turns out, it had nothing to do with BH.

I noticed my hosting filesystem had run out of inodes but it had plenty of space remaining, so I started searching around.

mcMMO decided it was cool to dump its block log records in every /world/mcmmo_data folder it could make. It was literally over 20k files and took about an hour to remove. Why these don't get put inside the mcMMO database, I do not know.

Regardless, after doing this there is no more hang time. Sorry for wasting your time.
